docker/provision/start-ralph.sh

```diff
@@ -0,0 +1,10 @@
+#!/bin/bash
+
+/opt/ralph/ralph-core/bin/gunicorn --daemon -c "${RALPH_LOCAL_DIR}/gunicorn.ini" ralph.wsgi
+
+while [ ! -f /var/log/ralph/ralph.log ] || [ ! -f /var/log/ralph/gunicorn.error.log ]
```

Comment:
Can we use ``wait-for-it.sh`` for that?

Reply:
Good question. Sounds like something we should do. Let me test that.

Reply:
Seems to work